<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>gLinDA 1.0</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ConfigTab">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <layout class="QVBoxLayout" name="ConfigP2PLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="P2PLabel">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Peer-to-Peer Network</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="HostLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="HostInput"/>
        </item>
        <item>
         <widget class="QLabel" name="PeersLabel">
          <property name="text">
           <string>Peers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Peer1Input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="Peer2Input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="Peer3Input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="Peer4Input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="Peer5Input">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PasswordLabe">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PasswordInput"/>
        </item>
        <item>
         <widget class="QLabel" name="EncryptionLabel">
          <property name="text">
           <string>Encryption</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="EncryptionGroup">
          <item>
           <widget class="QRadioButton" name="RSAEncryption">
            <property name="text">
             <string>RSA+AES</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="AESEncryption">
            <property name="text">
             <string>AES</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="SoloMode">
          <property name="text">
           <string>Single Mode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="ConfigLINDALayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>35</number>
        </property>
        <item>
         <widget class="QLabel" name="LindaLabel">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>LinDA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="CovariatesLabel">
          <property name="text">
           <string>Linear Formula</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="CovariatesInput"/>
        </item>
        <item>
         <widget class="QLabel" name="DataTablesLabel">
          <property name="text">
           <string>Data Tables</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="FeatureDataTableButton">
          <property name="text">
           <string>Feature Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="MetaDataTableButton">
          <property name="text">
           <string>Meta Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="FeatureDataTypeLabel">
          <property name="text">
           <string>Feature Data Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FeatureDataTypeCombo">
          <item>
           <property name="text">
            <string>count</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>proportion</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AlphaLabel">
          <property name="text">
           <string>Alpha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AlphaInput">
          <property name="text">
           <string>0.05</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="WinsorLabel">
          <property name="text">
           <string>Winsor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="WinsorCombo">
          <item>
           <property name="text">
            <string>Yes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>No</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="OutlierPctLabel">
          <property name="text">
           <string>Expected Outlier Percentage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="OutlierPctInput">
          <property name="text">
           <string>0.03</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Logo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>logo.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="ConfigLINDA2Layout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>126</number>
        </property>
        <item>
         <widget class="QLabel" name="ZeroHandlingLabel">
          <property name="text">
           <string>Zero Handling</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ZeroHandlingInput">
          <item>
           <property name="text">
            <string>adaptive</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pseudo_count</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>imputation</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="ResultTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextBrowser" name="ResultBrowser">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="BottomBar">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="MessageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Empty</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="ProgressBar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RunButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Run!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenConfig"/>
    <addaction name="actionSaveConfig"/>
    <addaction name="actionExportConfig"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="FileMenu"/>
  </widget>
  <action name="actionOpenConfig">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open configuration</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Close gLinDA</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExportConfig">
   <property name="text">
    <string>Export...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpenConfig</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_configuration_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HostInput</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PasswordInput</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Peer1Input</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Peer2Input</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Peer3Input</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Peer4Input</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveConfig</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportConfig</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>export_config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RSAEncryption</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AESEncryption</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SoloMode</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>solo_mode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AlphaInput</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>check_run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RunButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FeatureDataTableButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_feature_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MetaDataTableButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_metadata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_configuration_file()</slot>
  <slot>check_run_btn()</slot>
  <slot>run_btn()</slot>
  <slot>save_config()</slot>
  <slot>export_config()</slot>
  <slot>solo_mode()</slot>
  <slot>select_feature_data()</slot>
  <slot>select_metadata()</slot>
 </slots>
</ui>
